frontend/packages/dev-console/src/utils/imagestream-utils.ts

```diff
@@ -212,3 +191,16 @@ export const getProjectResource = (): FirehoseResource[] => {
     },
   ];
 };
+
+export const getImageStreamResource = (namespace: string): FirehoseResource[] => {
+  const resource = [];
+  if (namespace && namespace !== '') {
```

Comment:
`namespace` cannot be truthy while `namespace` is equal to an empty string... this is a duplicate check.
```suggestion
  if (namespace) {
```